### PR TITLE
set up alex linting

### LINKED ISSUE #48: Set up alex bot
https://github.com/get-alex/alex

### PR DESCRIPTION
**Pull request by artmsilva**
Run `npm run alex`

https://alexjs.com/

Arturo Silva 2022-03-08 at 09 52 44

Resolves #48

### CODE REVIEW

File: docs/components/button.mdx [hunk 1]
```diff
@@ -279,7 +279,7 @@ export default function Example() {
 
 ## Behavior
 
-### Disabled
+### Turned off
```
Comment (BrianAA) [inline docs/components/button.mdx]: Disabled has been changed to Turned off even though we added it to allow configuration?

File: docs/components/checkbox.mdx [hunk 2]
```diff
@@ -176,7 +176,7 @@ export default function Example() {
 
 ## Behavior
 
-### Disabled
+### Turned off input
```
Comment (BrianAA) [inline docs/components/checkbox.mdx]: I worry this might be confusing if we are really moving away from disabled

File: docs/foundations/color.mdx [hunk 3]
```diff
@@ -36,7 +36,7 @@ When using color tokens in code, you can access them using
 
 ## Static color tokens
 
-Static color tokens remain static regardless of the current context. To use a static token apply a -static prefix to the color token.
+Static color tokens hold their value regardless of the current context. To use a static token apply a -static prefix to the color token.
```
Comment (BrianAA) [inline docs/foundations/color.mdx:39]: interesting...I wonder what issue was here? Why the change

File: docs/foundations/principles.mdx [hunk 4]
```diff
@@ -60,7 +60,7 @@ Use color to create an [affordance](https://www.merriam-webster.com/dictionary/a
 
 #### System status
 
-`Semantic colors` represent system status of default, success, or failed states when completing an action or goal.
+`Semantic colors` represent system status of default, success, or faulty states when completing an action or goal.
```
Comment (BrianAA) [inline docs/foundations/principles.mdx]: Faulty and failed feel like two different things. Something faulty feels broken a failed state is in an intended state of an object.

File: docs/resources/guides/contribute-a-component.mdx [hunk 5]
```diff
@@ -89,9 +89,9 @@ When developing a component, use these core tenets for component design:
 
 2. Now that you have reviewed the specs and the JIRA ticket, please reach out to the Core team on our Slack channel. We're review the specs and the JIRA ticket and will assign the ticket to you.
 
-3. You are now ready to start contributing. Review the component design resource post for how we design components. Now that you have reviewed our component design architecture, open the codebase up and review the contributing guidelines. We have a simple way to stub out a new component using some boilerplate code (`npm run new-component <component-name>`).
+3. You are now ready to start contributing. Review the component design resource post for how we design components. Now that you have reviewed our component design architecture, open the codebase up and review the contributing guidelines. We have a way to stub out a new component using some boilerplate code (`npm run new-component <component-name>`).
```
Comment (BrianAA) [inline docs/resources/guides/contribute-a-component.mdx:92]: I can def appreciate the reducing of words like simple

**Review by BrianAA — COMMENTED** [review]
Love what this is attempting to accomplish. Does the linter create suggestions to be reviewed and then for you to change it?

File: docs/resources/guides/contribute-a-component.mdx (hunk 5 above)
Comment (artmsilva) [reply docs/resources/guides/contribute-a-component.mdx:92]: Why they recommend not using "simple" https://github.com/retextjs/retext-equality/blob/main/data/en/condescending.yml#L24

File: docs/foundations/principles.mdx (hunk 4 above)
Comment (artmsilva) [reply docs/foundations/principles.mdx]: I can add an exception.

File: docs/components/checkbox.mdx (hunk 2 above)
Comment (artmsilva) [reply docs/components/checkbox.mdx]: Maybe I can put it inside a tick `disabled`

File: docs/components/button.mdx (hunk 1 above)
Comment (artmsilva) [inline docs/components/button.mdx]:
```suggestion
### Disabled
```

File: docs/components/checkbox.mdx (hunk 2 above)
Comment (artmsilva) [inline docs/components/checkbox.mdx]:
```suggestion
### Disabled
```

File: docs/foundations/principles.mdx (hunk 4 above)
Comment (artmsilva) [inline docs/foundations/principles.mdx]:
```suggestion
`Semantic colors` represent system status of default, success, or failed states when completing an action or goal.
```

File: .alexrc [hunk 6]
```diff
@@ -0,0 +1,3 @@
+{
+  "allow": ["color", "colors", "disabled", "Disabled", "remain"]
```
Comment (artmsilva) [inline .alexrc]:
```suggestion
  "allow": ["color", "colors", "disabled", "Disabled", "remain", "failed"]
```

File: docs/foundations/principles.mdx [hunk 7]
```diff
@@ -204,7 +204,7 @@ assigns a color intention.
           color: "$onDisabled"
         }}
       >
-        Disabled
+        Turned off
```
Comment (artmsilva) [inline docs/foundations/principles.mdx]:
```suggestion
        Disabled
```

**Review by BrianAA — APPROVED** [review]
Love it!
